### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/names`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4573]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4573|
|Type|enhancement|
|Created|2010-07-11 07:02:50Z|
|Branch|https://github.com/twisted/twisted/tree/sphinx-news-4573|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for `projects/names`

Attachments:

* [names.xhtml.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/68c/68cb70ea21b8a3528aee58384ab83ddfbbcda02b/2a99c4f2abc7d2e27de3bb5b214bf8df94ba6b2f.patch) (666 bytes) - added by jdb on 2010-09-09 17:11:26Z - I modified the file names.html to make sure the <code> tag is not at the beginning of a line. No more missing space in the rst between a word and "``" either or **.

<details><summary>Searchable metadata</summary>

```
trac-id__4573 4573
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_sphinx_news_4573 branches/sphinx-news-4573
branch_author__glyph glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1278831770000000 1278831770000000
changetime__1284570330000000 1284570330000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="jdb's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|jdb removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

(In [30014]) Branching to 'sphinx-news-4573'

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @glyph|
|-|-|

As per the difference between http://buildbot.twistedmatrix.com/builds/sphinx-html-15660/projects/names/howto/names.html and http://buildbot.twistedmatrix.com/builds/sphinx-html-15662/projects/names/howto/names.html this looks good.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph set status to `closed`|
|-|-|

(In [30018]) Tidied up the Twisted Names documentation for easier conversion.

Author: jdb
Reviewer: glyph

Fixes [#4573](https://github.com/twisted/twisted/issues/4573)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
